growth/too/models.py

```diff
@@ -49,6 +51,8 @@ def create_all():
     db.create_all(bind=None)
 
     telescopes = ["ZTF", "Gattini", "DECam", "KPED", "GROWTH-India"]
+    telescopes = ["ZTF", "DECam"]
+    telescopes = ["ZTF"]
```

Comment:
Uh oh. You deleted all telescopes except ZTF!

Reply:
oops

Reply:
Fixed.